analizator.py

```diff
@@ -108,4 +109,13 @@ def countSentences(text):
     elif action == 7:
         plik = open('statystyki.txt', 'w')
         plik.write("2: %s\n3: %s \n4: %s \n5: %s" % (countLetters(text, False), countWords(text), countPunctations(text), countSentences(text)))
-        plik.close()
\ No newline at end of file
+        plik.close()
+    elif action == 8:
+        g = input("Exit program and delete files? Type 'y' to proceed or any key to cancel\n")
+        if g == "y":
+            os.remove("statystyki.txt")
+            os.remove("6.txt" )
+            print("Files Removed!")
+            exit()
+        else:
+            print ('doing nothing')
```

Comment:
This is not proper implementation. Please change it so it won't crash when files do not exists.